### PR TITLE
Added requirements.txt and requirements_dev.txt

### PR DESCRIPTION
**Pull request by polyglot-jones**
which represent the modules I had to install in order to get the unit tests to run for me.

### CODE REVIEW

**Comment by goldstar611** [comment]
Hi,

Did you generate the requirements.txt using pip freeze? I don't think that `pywinauto` or `pywin32` can be installed on either macOS or Linux operating systems and I have not seen CHIRP reference either package before.

**Comment by kk7ds** [comment]
pywin32 won't be installable on other platforms, so please remove that. Or maybe just document it elsewhere?

**Comment by polyglot-jones** [comment]
Ah, gotchya. Done.

File: run_all_tests.bat [hunk 1]
```diff
@@ -1,3 +1,3 @@
 @echo off
-python tests\run_tests.py
-python tools\cpep8.py
```
Comment (kk7ds) [inline run_all_tests.bat:3]: Did you mean to make these changes? I don't see them mentioned in the commit message, so maybe they were accidental? I'm not a windows user, so I don't know what "py -2" does (aside from the obvious guess). Is that definitely a thing for people with python 2.7 installed?

Comment (polyglot-jones) [reply run_all_tests.bat:3]: Oh, frack. Yeah, that was just for me. (FYI: I think I actually have a better way now, going thru a VM that has only Python2 installed.)

**Pull request closed without merging**